app/models/patient.rb

```diff
@@ -8,14 +8,14 @@ class Patient < ApplicationRecord
                     tsearch: { prefix: true }
                   }
 
-  enum gender: { masculino: 0, femenino: 1 }
-  enum marital_status: { casado: 0, divorciado: 1, soltero: 2, union_libre: 3, viudo: 4 }
+  enum gender: { m: 0, f: 1 }
```

Comment:
Actually it is ok having the whole symbol. This line will generate one helper per symbol. And having thins like this just make no sence:
```
braulio = Patient.find(1)
braulio.m?
braulio.f?
braulio.s?
braulio.c?
braulio.u_l?
```

Reply:
Let's roll this back is it was. If you need some particular text on the views, add the required things on a helper or directly on the html tags

Reply:
Reversed the changes, Should I edit the helpers to only show M, F, etc. or just display the whole word?

Reply:
Document says it should only be `M` and `F`, so answers is yes

Reply:
Same with the `marital_status`